source/data-modeling/field-types.txt

```diff
@@ -0,0 +1,581 @@
+as symbols to {+language+} applications. ``StringifiedSymbol`` allows you to use symbols
+while ensuring interoperability with other drivers. This type stores all data on
+the database as strings, and converts the strings to symbols when read by the
+application. Values that cannot be directly converted to symbols, such as
+integers and arrays, are converted into strings and then into symbols.
+
+The following example stores a ``status`` field as a ``StringifiedSymbol`` and
+demonstrates how the field is stored and returned:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-stringified-symbol
+   :end-before: # end-stringified-symbol
+
+Specify Field Types as Strings or Symbols
+-----------------------------------------
+
+You can use strings or symbols to specify certain field types in {+odm+}, instead of
+using their class names. The following example specifies the ``state`` field by
+using the class name, a string, and a symbol:
+
+.. code-block:: ruby
+
+   class Order
+     include Mongoid::Document
+     
+     # Class Name
+     field :order_num, type: Integer
+
+     # Symbol
+     field :order_num, type: :integer
+     
+     # String
+     field :order_num, type: "integer"
+   end
+
+The following table provides the field types that can you can specify as strings or symbols:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Class Name
+     - Symbol
+     - String
+
+   * - ``Array``
+     - ``:array``
+     - ``"Array"``
+   
+   * - ``BigDecimal``
+     - ``:big_decimal``
+     - ``"BigDecimal"``
+   
+   * - ``BSON::Binary``
+     - ``:binary``
+     - ``"BSON::Binary"``
+   
+   * - ``Mongoid::Boolean``
+     - ``:boolean``
+     - ``"Mongoid::Boolean"``
+   
+   * - ``Date``
+     - ``:date``
+     - ``"Date"``
+   
+   * - ``DateTime``
+     - ``:date_time``
+     - ``"DateTime"``
+   
+   * - ``Float``
+     - ``:float``
+     - ``"Float"``
+   
+   * - ``Hash``
+     - ``:hash``
+     - ``"Hash"``
+   
+   * - ``Integer``
+     - ``:integer``
+     - ``"Integer"``
+   
+   * - ``BSON::ObjectId``
+     - ``:object_id``
+     - ``"BSON::ObjectId"``
+   
+   * - ``Range``
+     - ``:range``
+     - ``"Range"``
+   
+   * - ``Regexp``
+     - ``:regexp``
+     - ``"Regexp"``
+   
+   * - ``Set``
+     - ``:set``
+     - ``"Set"``
+   
+   * - ``String``
+     - ``:string``
+     - ``"String"``
+   
+   * - ``StringifiedSymbol``
+     - ``:stringified_symbol``
+     - ``"StringifiedSymbol"``
+   
+   * - ``Symbol``
+     - ``:symbol``
+     - ``"Symbol"``
+   
+   * - ``Time``
+     - ``:time``
+     - ``"Time"``
+
+Custom Field Types
+------------------
+
+You can create custom field types and define how {+odm+} serializes and
+deserializes them. To create a custom field type, define a class that
+implements the following methods:
+
+- ``mongoize``, which takes an instance of your custom type and converts it to
```

Comment:
S: consider changing the format of this list to match whats done in other sections
```suggestion
- ``mongoize``: Takes an instance of your custom type and converts it to
```